cli/ceph/orch/host.py

```diff
@@ -24,3 +27,24 @@ def ls(self, **kw):
         if isinstance(out, tuple):
             return out[0].strip()
         return out
+
+    def maintenance(self, hostname, operation, **kw):
+        """
+        Add/Remove the specified host from maintenance mode
+        Args:
+            hostname(str): name of the host which needs to be added into maintenance mode
+            operation(str): enter/exit maintenance mode
+            kw (dict): Key/value pairs that needs to be provided to the installer
+
+        Supported keys:
```

Comment:
`**kw` is used for large no of parameters. For simplicity pass `force` & `yes_i_really_mean_it` parameters to method.

Reply:
Done